Engine/Src/Ancona/Util2D/Collision/Box2.cpp

```diff
@@ -42,13 +78,32 @@ void Box2::GetVertices(std::vector< std::pair<float,float> > & vertices) const
 
 bool Box2::Intersects(const Box2 & box) const
 {
-    sf::Vector2f fixVector;
-    float mag;
-    return Intersects(box, fixVector, mag);
+    auto intersects = OptimizedIntersect(*this, box);
+    if (intersects == DoesIntersect::Maybe)
```

Comment:
Why is this one == DoesIntersect::Maybe but the override of Intersects checks for == DoesIntersect::No?